kong/clustering/wrpc_data_plane.lua

```diff
@@ -82,20 +75,23 @@ local function init_config_service(service)
       end
     end
     return accept_table
-  end)
-end
-
-local wrpc_services
-local function get_services()
-  if not wrpc_services then
-    wrpc_services = wrpc_proto.new()
-    init_negotiation_client(wrpc_services)
-    init_config_service(wrpc_services)
   end
 
-  return wrpc_services
+  get_services = function()
+    if not wrpc_services then
+      wrpc_services = wrpc_proto.new()
+
+      wrpc_services:import("kong.services.config.v1.config")
+      wrpc_services:set_handler("ConfigService.SyncConfig", config_handler)
+
+      init_negotiation_client(wrpc_services)
```

Comment:
It might be OK to put `nego` service initialization after `conf` service, but they were intentionally ordered as nego happens before conf.

Reply:
Just for readability. It's fine for both orders.